### ISSUE TITLE
SC 3.3.8 Redundant Entry is missing the orange "Status" warning

### ISSUE THREAD

**Issue by StommePoes** [w3c/wcag#1277]
The [Understanding page](https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html) is a new proposed SC but does not have

>Status
>This understanding document is part of the draft WCAG 2.2 content. It may change or be removed before the final WCAG 2.2 is published.
